k8s-samples/existing-lighthouse/openshift/k8-lighthouse.yaml

```diff
@@ -0,0 +1,170 @@
+---
+apiVersion: storage.k8s.io/v1beta1
+kind: StorageClass
+metadata:
+  name: portworx-sc
+provisioner: kubernetes.io/portworx-volume
+parameters:
+  repl: "3"
+
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: pvcsc001
+  annotations:
+    volume.beta.kubernetes.io/storage-class: portworx-sc
+  creationTimestamp: null
+  labels:
+    service: influx-px-claim0
+  name: influx-px-claim0
+spec:
+  accessModes:
+  - ReadWriteOnce
+  resources:
+    requests:
+      storage: 2Gi
+status: {}
+
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  creationTimestamp: null
+  labels:
+    service: influx-px
+  name: influx-px
+spec:
+  replicas: 1
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      creationTimestamp: null
+      labels:
+        service: influx-px
+    spec:
+      containers:
+      - env:
+        - name: MYSQL_ALLOW_EMPTY_PASSWORD
+          value: "true"
+        - name: ADMIN_USER
+          value: '"admin"'
+        - name: INFLUXDB_INIT_PWD
+          value: '"password"'
+        - name: PRE_CREATE_DB
+          value: '"px_stats"'
+        image: tutum/influxdb
+        name: influx-px
+        ports:
+        - containerPort: 8083
+        - containerPort: 8086
+        resources: {}
+        volumeMounts:
+        - mountPath: /data
+          name: influx-px-claim0
+      restartPolicy: Always
+      volumes:
+      - name: influx-px-claim0
+        persistentVolumeClaim:
+          claimName: influx-px-claim0
+status: {}
+
+---
+apiVersion: v1
+kind: Service
+metadata:
+  creationTimestamp: null
+  labels:
+    service: influx-px
+  name: influx-px
+spec:
+  ports:
+  - name: "8083"
+    port: 8083
+    targetPort: 8083
+  - name: "8086"
+    port: 8086
+    targetPort: 8086
+  selector:
+    service: influx-px
+status:
+  loadBalancer: {}
+
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  creationTimestamp: null
+  labels:
+    service: px-lighthouse
+  name: px-lighthouse
+spec:
+  replicas: 1
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      creationTimestamp: null
+      labels:
+        io.kompose.service: px-lighthouse
+    spec:
+      containers:
+      - command:
+        - /bin/bash
+        - /lighthouse/on-prem-entrypoint.sh
+        - -k
+        - etcd:http://<etcd server>:<ETCD PORT>
+        - -d
+        - http://admin:password@influx-px:8086
+        env:
+        - name: PWX_INFLUXDB
+          value: '"http://influx-px:8086"'
+        - name: PWX_INFLUXUSR
+          value: '"admin"'
+        - name: PWX_INFLUXPW
+          value: '"password"'
+        - name: PWX_HOSTNAME
+        - name: PWX_PX_PRECREATE_ADMIN
+          value: "true"
+        - name: PWX_PX_COMPANY_NAME
+          value: <COMPANY NAME>
+        - name: PWX_PX_ADMIN_EMAIL
+          value: <ADMIN EMAIL>
```

Comment:
Here I would have used the quotes around the value, as your current demo value does contain the space.

Just a suggestion, perhaps it would be more intuitive to use a standard prefix on all the values that will require changing (e.g. `MY_COMPANY_NAME` and `MY_ADMIN_EMAIL`)